src/app/courses/manage-course/course-manage.component.ts

```diff
@@ -0,0 +1,25 @@
+import { Component, OnInit, OnDestroy } from '@angular/core';
+import { ActivatedRoute } from '@angular/router';
+
+@Component({
+  templateUrl: './course-manage.component.html'
+})
+export class CourseManageComponent implements OnInit, OnDestroy {
+  id: string;
+  private sub: any;
+
+  constructor(private route: ActivatedRoute) {}
+
+  ngOnInit() {
+    this.sub = this.route.params.subscribe(params => {
+       this.id = params['id']; // (+) converts string 'id' to a number
+
```

Comment:
Removed unncessary comments/spaces/Extra line. Could you please check this one too ?